scripts/db_migrator.py

```diff
@@ -473,7 +473,22 @@ def version_2_0_0(self):
         Current latest version. Nothing to do here.
         """
         log.log_info('Handling version_2_0_0')
+        warmreboot_state = self.stateDB.get(self.stateDB.STATE_DB, 'WARM_RESTART_ENABLE_TABLE|system', 'enable')
+
+        if warmreboot_state != 'true':
+            portchannel_table = self.configDB.get_table('PORTCHANNEL')
+            for name, data in portchannel_table.items():
+                data['lacp_key'] = 'auto'
```

Comment:
Shouldn't we do a configDB.get_entry() first for that key and update the 'lacp_key' alone as a new field and then set_entry(). Otherwise the other valid fields already present would be removed.

Reply:
@judyjoseph , why can't we treat `lacp_key` to `auto `if not specified?.

Reply:
We kept the old behavior if lacp_key is not set to make sure that if we are upgrading from an older version and doing warm reboot the lacp key won't change because otherwise it will cause a drop in the connection.

Reply:
@judyjoseph we are already getting the keys information when we are calling the portchannel_table.items() and retrieving name and data.
The other field are present in the data variable and there is no need for calling another function to retrieve them.